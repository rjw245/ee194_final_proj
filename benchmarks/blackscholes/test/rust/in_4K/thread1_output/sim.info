{'cmdline': ['/h/rwood04/ee194/sniper/run-sniper',
             '-d',
             './thread1_output',
             '-n',
             '1',
             '-c',
             'gainestown',
             '-c',
             'nehalem.cfg',
             '--',
             '../../rust/bs',
             '1',
             '../../inputs/in_4K.txt'],
 'git_revision': '',
 'host': 'dell26',
 'pin_version': 71313,
 'rusage': (817.969649,
            2.7455819999999997,
            92064,
            0,
            0,
            0,
            369305,
            0,
            0,
            0,
            11200,
            0,
            0,
            0,
            98449,
            185814),
 'snipercmd': ['bash',
               '-c',
               '/h/rwood04/ee194/sniper/pin_kit/intel64/bin/pinbin -mt -injection child -xyzzy -enable_vsm 0 -t /h/rwood04/ee194/sniper/lib/pin_sim -c /h/rwood04/ee194/sniper/config/base.cfg --general/total_cores=1 --general/output_dir=/h/rwood04/ee194/ee194_docs/final_project/ee194_final_proj/benchmarks/blackscholes/test/rust/thread1_output --config=/h/rwood04/ee194/sniper/config/nehalem.cfg --config=/h/rwood04/ee194/sniper/config/gainestown.cfg --config=/h/rwood04/ee194/sniper/config/nehalem.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -- ../../rust/bs 1 ../../inputs/in_4K.txt'],
 't_elapsed': 1174.1363098621368,
 't_start': 1460948560.295808,
 'tracecmds': [],
 'user': 'rwood04',
 'vmem': 3181137920L}
