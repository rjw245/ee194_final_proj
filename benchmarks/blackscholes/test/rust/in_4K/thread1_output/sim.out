                                     | Core 0    
  Instructions                       |  283095661
  Cycles                             |  167340866
  IPC                                |       1.69
  Time (ns)                          |   62910100
  Idle time (ns)                     |       1151
  Idle time (%)                      |       0.0%
Branch predictor stats               |           
  num correct                        |   10008427
  num incorrect                      |     457316
  misprediction rate                 |      4.37%
  mpki                               |       1.62
TLB Summary                          |           
  I-TLB                              |           
    num accesses                     |   57350902
    num misses                       |        194
    miss rate                        |      0.00%
    mpki                             |       0.00
  D-TLB                              |           
    num accesses                     |  187826265
    num misses                       |       2906
    miss rate                        |      0.00%
    mpki                             |       0.01
  L2 TLB                             |           
    num accesses                     |       3100
    num misses                       |        428
    miss rate                        |     13.81%
    mpki                             |       0.00
Cache Summary                        |           
  Cache L1-I                         |           
    num cache accesses               |   57350902
    num cache misses                 |    1288635
    miss rate                        |      2.25%
    mpki                             |       4.55
  Cache L1-D                         |           
    num cache accesses               |  187826549
    num cache misses                 |      16914
    miss rate                        |      0.01%
    mpki                             |       0.06
  Cache L2                           |           
    num cache accesses               |    1305661
    num cache misses                 |      12064
    miss rate                        |      0.92%
    mpki                             |       0.04
  Cache L3                           |           
    num cache accesses               |      12183
    num cache misses                 |      10625
    miss rate                        |     87.21%
    mpki                             |       0.04
DRAM summary                         |           
  num dram accesses                  |      10620
  average dram access latency (ns)   |      70.65
  average dram queueing delay        |      17.22
  average dram bandwidth utilization |      0.00%
Coherency Traffic                    |           
  num loads from dram                |       3457
  num loads from dram cache          |          0
  num loads from remote cache        |          0
